common/lib/xmodule/xmodule/js/src/video/06_video_progress_slider.js

```diff
@@ -147,25 +149,6 @@ function () {
         // with actual starting and ending point of the video.
 
         rangeParams = getRangeParams(start, end, duration);
-
-        if (!this.videoProgressSlider.sliderRange) {
```

Comment:
I'm unfamiliar with the section; what is the `sliderRange` and why are we removing this?